src/utils/formErrorUtil.ts

```diff
@@ -14,11 +14,7 @@ export const combineValidation = (validations: Array<IValidation>) => (value: st
 
 export const required: IValidation = (value: string) => validation.isRequired(value) ? undefined : 'Field is required';
 export const email: IValidation = (value: string) => validation.isEmail(value) ? undefined : 'Wrong email';
-export const password: IValidation = (value: string) => validation.isPassword(value)
-  ? undefined : 'Bad characters password';
-export const date: IValidation = (value: string) => validation.isFuture(value)
-  ? undefined : 'You can not to be born in the future';
-export const age: IValidation = (value: string) => validation.isAdult(value)
-  ? undefined : 'You are not adult';
-export const passwordRequirements: IValidation = (value, newValue) => validation.passwordMatching(value, newValue)
-  ? undefined : 'Passwords do not match';
+export const password: IValidation = (value: string) =>
+  validation.isPassword(value) ? undefined : 'Bad characters password';
+export const passwordRequirements: IValidation = (value, newValue) =>
+  validation.passwordMatching(value, newValue) ? undefined : 'Passwords do not match';
```

Comment:
ohhhh
all these errors should be in translations with using i18next
pay attention to withFormField implementation